ktlint-ruleset-template/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    `ktlint-kotlin-common` // replace it with 'org.jetbrains.kotlin.jvm'
+    id("ktlint-kotlin-common")
```

Comment:
Was this `` ` -> id("") `` because Gradle doesn't generate the magic any more? For consistency would it make sense to replace the other backtick plugins with strings too?

Reply:
Yeah, will generate typesafe id with buildSrc.